runtime-sdk/modules/evm/src/derive_caller.rs

```diff
@@ -0,0 +1,44 @@
+use tiny_keccak::{Hasher, Keccak};
```

Comment:
As mentioned before, can we switch to the `sha3` crate as it is used elsewhere?

Reply:
filed #497